Base/usr/share/man/man1/realsize.md

```diff
@@ -0,0 +1,40 @@
+## Name
+
+realsize - Show the 'real' size of a number with a suffix (possibly).
```

Comment:
Would it be better to be a bit more explicit about what the utility does here? Something like "Print the size of a number with a suffix, in bytes."